test/sql/alter.sql

```diff
@@ -205,6 +205,30 @@ SELECT tablename, tablespace FROM pg_tables
 WHERE tablename = 'hyper_in_space' OR tablename ~ '_hyper_\d+_\d+_chunk' ORDER BY tablename;
 SELECT * FROM _timescaledb_catalog.tablespace;
 
+-- tablespace functions should handle the default tablespace just as they do others
+SELECT attach_tablespace('pg_default', 'hyper_in_space');
```

Comment:
If I'm reading the code correctly, users should be able set the tablespace to the database's default even if the hypertable's owner no longer has permissions for that tablespace. It would be nice if we could test that case as well, but not urgent.